application/controllers/ConfigController.php

```diff
@@ -370,7 +370,7 @@ public function diffAction()
 
         $configs = $db->enumDeployedConfigs();
         foreach (array($leftSum, $rightSum) as $sum) {
-            if (! array_key_exists($sum, $configs)) {
+            if ($sum && ! array_key_exists($sum, $configs)) {
```

Comment:
Note:
 Phpstan failed to detect this:  substr(): Passing null to parameter#1 ($string) of type string is deprecated

This is a mystery to me, because 2 days ago phpstan showed me the error.